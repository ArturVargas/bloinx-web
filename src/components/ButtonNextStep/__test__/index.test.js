import React from 'react';
import { IntlProvider } from 'react-intl';
import renderer from 'react-test-renderer';

import { ButtonNextStep } from '../index';

describe('ButtonNextStep', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <IntlProvider
        locale="en"
        onError={(err) => {
          if (err.code === 'MISSING_TRANSLATION') {
            return;
          }
          throw err;
        }}
      >
        <ButtonNextStep intl={{}} onBack={jest.fn()} onClick={jest.fn()} disabled />
      </IntlProvider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
