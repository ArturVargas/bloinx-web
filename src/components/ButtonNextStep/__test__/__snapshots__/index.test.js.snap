// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonNextStep renders correctly 1`] = `
<div
  className="ButtonNextStep"
>
  <button
    className="ant-btn"
    onClick={[Function]}
    type="button"
  >
    commons.buttons.back
  </button>
  <button
    className="ant-btn"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    commons.buttons.continue
  </button>
</div>
`;
